ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -454,4 +452,9 @@ TokenListItem.propTypes = {
    * address represents the token address
    */
   address: PropTypes.string,
+
+  /**
+   * hidePercentage represents if the increase decrease percentage will be hidden
+   */
+  hidePercentage: PropTypes.bool,
```

Comment:
Rather than use negative language `hide`, we should stay with positive (`show`)

Reply:
done